### PR TITLE
Add Honeycomb Exporter to the Website

### PR DESCRIPTION
**Pull request by akvanhar**
The Honeycomb exporter is available [here](https://github.com/honeycombio/opencensus-exporter).

### CODE REVIEW

File: content/guides/exporters/supported-exporters/Go/Honeycomb.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+- Create an exporter in code
+- Pass in your Honeycomb API key, found on your Honeycomb Team Settings page. ([Sign up for free](https://ui.honeycomb.io/signup) if you haven’t already!)
+- Pass in your Honeycomb dataset name
+
+{{<highlight go>}}
+package main
+
+import (
+"log"
```
Comment (odeke-em) [inline content/guides/exporters/supported-exporters/Go/Honeycomb.md]: Please gofmt these snippets code :)

**Comment by songy23** [comment]
Please sign the CLA.

**Comment by akvanhar** [comment]
I've forwarded the CLA to the appropriate team member. I'll make sure to update once it's signed.

**Comment by rakyll** [comment]
I am also adding the exporter to the Go README, see https://github.com/census-instrumentation/opencensus-go/pull/925.

**Comment by akvanhar** [comment]
I've signed the CLA, and verified that the email address used in the CLA (avanahrdenberg@honeycomb.io) is the same as the git commits.

**Pull request merged**